common/v2/features/SendAssets/components/TransactionFormData.tsx

```diff
@@ -90,13 +90,16 @@ export default function SendAssetsForm({
                 <div className="input-group-header">{translate('X_ADDRESS')}</div>
                 <Field
                   name="account"
+                  value={console.log(values.account)}
+                  // => {updateState({ transactionFields: { account: values.account } })
                   component={({ field, form }: FieldProps) => (
                     <AccountDropdown
                       name={field.name}
                       value={field.value}
-                      onChange={(option: IExtendedAccount) =>
-                        form.setFieldValue(field.name, option)
-                      }
+                      onChange={(option: IExtendedAccount) => {
+                        updateState({ transactionFields: { account: option } });
```

Comment:
Aren't we trying to avoid using this?

Reply:
I'm unsure if we are. What are we trying to use then?

Reply:
I think we were just trying to use the formik values as a single source of truth.

Reply:
So instead of updating state on change, we update state on formik submit?

Reply:
yup!

Reply:
To be done when field handling in completed